packages/frontend/web-ui/src/scss/home/_gameList.scss

```diff
@@ -0,0 +1,34 @@
+@use '../base/variables' as variables;
+
+.game-list-item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--cg-bkg-primary-light);
+  border: 2px solid var(--cg-bkg-secondary-light);
+  border-radius: 10px;
+  box-shadow: 0 2px 4px 0 rgba(68, 2, 131, 0.5);
+  margin: 0.5rem;
+  padding: 1rem;
+}
+
+.game-list-item .game-list-item-text {
+  color: var(--cg-text-primary);
+  text-align: center;
+  font-size: variables.$font-sm;
+}
+
+.game-list-item .game-list-item-button {
+  background-color: var(--cg-bkg-primary-dark);
+  border: 1px solid var(--cg-bkg-secondary-light);
+  font-weight: bold;
+  height: 2.5rem;
+  width: 7rem;
+  box-shadow: 0 4px 8px 0 rgba(223, 192, 252, 0.5);
```

Comment:
Same, this is a new color, why is it being introduced?

Reply:
Done! The same in that in the last case, I've changed the color with a color of the our palette